firebase-dataconnect/src/test/kotlin/com/google/firebase/dataconnect/serializers/TimestampSerializerUnitTest.kt

```diff
@@ -75,6 +75,23 @@ class TimestampSerializerUnitTest {
       .isEqualTo(Timestamp(1136214245, 123_456_789))
   }
 
+  @Test
+  fun `decoding should succeed when time offset is 0`() {
+    assertThat(decodeTimestamp("2006-01-02T15:04:05-00:00")).isEqualTo(Timestamp(1136214245, 0))
```

Comment:
Rather than hardcoding the expected Timestamp, consider calculating the Timestamp with the equivalent UTC time and comparing it as equal.

For example, in this case, `decodeTimestamp("2006-01-02T15:04:05-00:00")` should return a `Timestamp` object that is equal to `decodeTimestamp("2006-01-02T15:04:05Z")` (the two strings are the same, _except_ for their time zone). Given that the `Z` time zone is verified to be correct by other tests, this makes the test more readable and also conveys that the time zone offsets is merely a convenient way to represent UTC time in a local time zone.